hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/cache/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Package for custom cache implementations
+ */
+package com.hazelcast.jet.sql.impl.cache;
```

Comment:
I'd put the class as an inner static class of `JsonPathUtil`. I'd say its behavior is quite unusual to be reused elsewhere.

If you think it might be reused, then it should go to the `hazelcast` module to the `com.hazelcast.internal.util` package. I'd also delete the `Cache` interface. But I leave the final decision up to you.

Reply:
Done